src/models.jl

```diff
@@ -17,3 +10,20 @@ struct BayesModel{Tp,Ti} <: AbstractBayesQuadModel{Tp,Ti}
     prior::Tp
     logintegrand::Ti
 end
+
+
+_prior(m::AbstractBayesQuadModel{<:MvNormal}) = m.prior
+_prior(m::AbstractBayesQuadModel) = MvNormal(ones(length(m.prior)))
+prior(m::AbstractBayesQuadModel) = exp ∘ logprior(m)
+logprior(m::AbstractBayesQuadModel) = x->logpdf(_prior(m), x)
```

Comment:
We have to change the quadrature part accordingly in case we change the prior to the function instead of the distribution (in the quadrature and `PriorSampling` they currently expects a distribution from `Distributions.jl`). We could also indicate in the name whether we mean the distribution or the function e.g. `_priord` and m.priord in case of a distribution from `Distributions.jl`

Reply:
Ah yeah you're right, I forgot to change this as well! I just wanted a more uniform interface between `prior` and `integrand`

Reply:
At least the tests are catching it :D